pupyl.py

```diff
@@ -0,0 +1,19 @@
+"""
```

Comment:
@92amartins, as a standalone application, maybe it could be better executing as an independent script, by adding:

```suggestion
#!/usr/bin/env python3
"""
```
as described in [Should I put #! (shebang) in Python scripts, and what form should it take?](https://stackoverflow.com/questions/6908143/should-i-put-shebang-in-python-scripts-and-what-form-should-it-take).

But maybe we should check if this shebang is OS agnostic (for example, it'll work on Windows?).

By doing this, we can rename the file from `pupyl.py` to just `pupyl`, and can call like this:

```shell
pupyl [OPTIONS] [ETC]
```
instead of
```shell
python pupyl.py [OPTIONS] [ETC]
```

Reply:
Tried to follow this recommendation and couldn't make that work on macOS. I use pyenv so maybe there's a trouble with shims. Can you make that work in linux?

Reply:
Yep, maybe it's something to do with `pyenv`, because I tested on Linux without issues. As Windows systems doesn't support shebangs, we should not worry about this system right now.

If you agree, we can focus to support Linux and MacOS for a while, trying to understand what's happening with `pyenv` on MacOS (unfortunately I don't have one right at hand).

Reply:
What you think @92amartins? Should we focus on Linux on this case?

Reply:
As we talked, we'll remove the shebang and will create a `shell` script passing parameters to the actual `cli`.